CHANGELOG.md

```diff
@@ -12,4 +12,4 @@ FEATURES:
 
 IMPROVEMENTS:
 
-* vApp metadata now is attached to the vapp rather to first vm in vapp.
+* vApp metadata is no longer added to first VM in vApp it will be added to vApp directly instead.
```

Comment:
nit: Maybe this one more like BugFix? @lvirbalas  what's your opinion?

Reply:
Then better to "BACKWARDS INCOMPATIBILITIES / NOTES" section, as this improvement is a breaking change after all.

Reply:
Fixed added it to "BACKWARDS INCOMPATIBILITIES / NOTES" section.